src/api/java/mekanism/api/providers/IModuleDataProvider.java

```diff
@@ -1,11 +1,15 @@
 package mekanism.api.providers;
 
+import java.util.Objects;
+import mekanism.api.MekanismAPI;
 import mekanism.api.gear.ICustomModule;
 import mekanism.api.gear.ModuleData;
 import net.minecraft.MethodsReturnNonnullByDefault;
 import net.minecraft.resources.ResourceLocation;
 
+@SuppressWarnings("removal")
 @MethodsReturnNonnullByDefault
+@Deprecated(forRemoval = true, since = "10.7.11")
```

Comment:
should this have a replace with comment?

Reply:
Unsure? Basically it is just use `Holder<ModuleData<?>>` or `DeferredHolder<ModuleData<?>, ...>` in places. There isn't a different thing that should be implemented in place of it.